docs/source/operations-guide/kubernetes.md

```diff
@@ -27,7 +27,7 @@ with a minimal configuration ready to work with Lumigator.
 
 > [!NOTE]If the Mistral and/or the OpenAI API is used, there are two ways to provide
 > it to Lumigator:
-> 
+>
```

Comment:
nit, extra space

Reply:
I think my PR is removing the space? Several docs never had the linter run over them, so you'll see this in several places. My changes are adding in the linting/formatting changes enforced by our pre-commit checks.